### ISSUE TITLE
Nested lists not displaying properly

### ISSUE THREAD

**Issue by StochasticSpring** [trade-informatics/caracal#68]
Hi, I had not had any problems with single level lists but when creating ordered list like so:

```
docx.ol do
  li 'Apple'
  li 'Banana' do
    ol do
      li 'Pie' do
        ol do
          li 'Hot'
          li 'Cold'
        end
      end
      li 'Bread'
    end
  end
  li 'Cherry'
end
```
and I open the resulting file in Word, I will get something like this: 

```
    1. Apple
    2. Banana
Pie
                       i. Hot
                       ii. Cold
Bread
    3. Cherry
```

I fired up the caracal-example server and downloaded a fresh sample, on opening in Word and scrolling to the lists section more or less gave me the same deal:
```
This is an ordered list with nested levels:
    1. This is the first item
    2. This is the second item
This is a nested item.
    3. This is a third item with a link
```
Coincidentally I notice that the unordered list is messed up in the same fashion. In this case I tried Word 2016 for Mac and on Windows.

Opening with Mac Pages, upload to Google doc and uploading to Office 365 is interesting - initially the nested list is displayed perfectly. However, once I click into the list-items and start hitting tab or shift-tab, the list-items move up and down in levels accordingly, but the new level formatting do not match up with those that are existing on the same level (e.g. indentation is all over the place). I also read in another issue that Word sometimes changes the XML on opening so I tried the above with generated files that I never opened and got the same results.

Lastly, I've compared the document.xml between a docx from Word itself with a working nested list and the Caracal generated one and found that the latter essentially conforms to the former - in any case I even monkey patched Caracal::Renderers::DocumentRender#render_listitem to emit xml which is exactly the same as what I saw in Word and it did not solve anything. The only other .xml I looked at was numbering.xml and they were entirely the same.

**Comment by jdugan** [comment]
Huh.  Thanks for the detailed report.  I've never noticed this behaviour myself, but truthfully, I don't use Caracal for work projects anymore (how's that for irony?).  So I'm not always the best judge of how subtle changes in Word affect Caracal.

Give me a couple days to finish up a work sprint, and then I'll circle back to this.

**Comment by jdugan** [comment]
I definitely see the same behaviour as you do.  I'll try to replicate your reverse-engineering check tonight and see if I notice any differences.

**Comment by jdugan** [comment]
It turned out to be a small change in the `numbering.xml` spec.  I had to explicitly declare that all the abstractNum definitions were hybrid multi-line definitions.  That single line corrected the behaviour for me.  Looks like everything was defaulting to single-line definitions without that.

Released v1.0.13
